data/extension/init/templates/rust/envoy.access_loggers/default/src/logger.rs

```diff
@@ -67,11 +62,8 @@ impl<'a> AccessLogger for SampleAccessLogger<'a> {
         // Update stats
         self.stats.log_entries_total().inc()?;
 
-        let now: DateTime<Local> = self.clock.now()?.into();
```

Comment:
this just increased download and compilation time. normal logging config will show the time.

Reply:
@mathetake what's your take on this.. should we consume the value of `self.clock.now()` in a cheaper way? Personally, I think we aren't required to use this in a strange way just to use it, and it is better off not using it at all.

Reply:
this shows the usage of `Clock` API in envoy-sdk, so maybe we'd better do that in a cheaper way (i.e. not intoing Datatime<Local>).

Reply:
fair enough!